tests/lib/pkgdb.sh

```diff
@@ -477,3 +558,18 @@ install_pkg_dependencies(){
     # shellcheck disable=SC2086
     distro_install_package $pkgs
 }
+
+# upgrade distribution and indicate if reboot is needed by outputting 'reboot'
+# to stdout
+distro_upgrade() {
+    case "$SPREAD_SYSTEM" in
+        arch-*)
+            if pacman -Syu --noconfirm 2>&1 | grep -q "there is nothing to do" ; then
```

Comment:
perhaps you could use MATCH here

Reply:
`MATCH` is only defined in the shell that runs the script defined in `spread.yaml` or `task.yaml`. If `pkgdb.sh` is used in separate shell, the function may not be defined.

Reply:
I have a solution for missing MATCH that came out of my refactoring. I have it ready but I'll only post it on Monday to let you guys rest too :)

Reply:
This margin is too small to contain it ;)?